python/lambda-ddb-mysql-etl-pipeline/app.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+from etl_pipeline_cdk.etl_pipeline_cdk_stack import EtlPipelineCdkStack
+import os, json
+
+PRE_PROD_ENV={
+	"region": "<region_id>",
+	"account": "<account_id>"
+}
+
+PROD_ENV={
+	"region": "<region_id>",
+	"account": "<account_id>"
+}
+
+STAGE=os.environ.get("AWS_CDK_ENV")
+
+with open(f"params-{STAGE}.json") as file_obj:
```

Comment:
Seems to fail here
``` shell
2020-01-18T00:52:38.5463314Z Traceback (most recent call last):
2020-01-18T00:52:38.5464150Z   File "app.py", line 19, in <module>
2020-01-18T00:52:38.5465233Z     with open(f"params-{STAGE}.json") as file_obj:
2020-01-18T00:52:38.5466320Z FileNotFoundError: [Errno 2] No such file or directory: 'params-None.json'
2020-01-18T00:52:38.6098692Z [31mSubprocess exited with error 1[39m
2020-01-18T00:52:44.7610111Z ##[error]Process completed with exit code 1
```

Reply:
for the submitter: When we have an example with a required environment variable it's generally best to add it as a context variable instead and update the cdk.json file to reflect a valid value for it.

Here is an example application that does such a thing

[Referencing a context variable](https://github.com/aws-samples/aws-cdk-examples/blob/e8d9b0427b8122aaff33b44005f4744ce9f31124/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/index.ts#L15-L17)

[Where the variable is set](https://github.com/aws-samples/aws-cdk-examples/blob/e8d9b0427b8122aaff33b44005f4744ce9f31124/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/cdk.json)